oauth2-authorization-server/src/test/java/org/springframework/security/oauth2/server/authorization/oidc/authentication/OidcClientRegistrationAuthenticationProviderTests.java

```diff
@@ -375,8 +375,9 @@ public void authenticateWhenClientRegistrationRequestAndValidAccessTokenThenRetu
 
 		// assert "registration" access token, which should be used for subsequent calls to client configuration endpoint
 		OAuth2Authorization authorizationResult = authorizationCaptor.getAllValues().get(0);
-		assertThat(authorizationResult.getAccessToken().getToken().getScopes())
-				.containsExactly("client.read");
+		Set<String> authorizedScopes = authorizationResult.getAccessToken().getToken().getScopes();
```

Comment:
Let's revert this change. Instead, configure the integration tests in `OidcClientRegistrationTests` to use `JdbcOAuth2AuthorizationService` instead of `InMemoryOAuth2AuthorizationService` and it will reproduce the error.

Reply:
@ovidiupopa91 Do you have time to apply this change by EOD tomorrow? I'd like to get this into this Wed's release. If you don't have time, I can apply it. Please let me know.

Reply:
hi @jgrandja. I had some problems with my laptop. It's working now. I will apply the change right now